build/yamls/windows/containerd-with-ovs/conf/PostStart-AntreaAgent-Containerd.ps1

```diff
@@ -0,0 +1,12 @@
+$networkName = "antrea-hnsnetwork"
```

Comment:
Do you think we can share one powershell script for lifecycle control, and use a parameter to decide enable or disable vmswitch extension? Then we can avoid duplicated code?

Reply:
done.

Reply:
The agent encountered an error log after this round's update：
```
 Warning  FailedPostStartHook  8s (x2 over 15s)  kubelet            FailedPostStartHook
  Warning  FailedPreStopHook    8s (x2 over 15s)  kubelet            Exec lifecycle hook ([powershell -file $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent-Containerd.ps1]) for Container "antrea-agent" in Pod "antrea-agent-windows-4srdg_kube-system(fa531e1e-93fc-41d6-ae6e-4808607191b4)" failed - error: command 'powershell -file $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent-Containerd.ps1' exited with 1: At C:\C\antrea-agent\var\lib\antrea-windows\VMSwitchExtension-Antre
aAgent-Containerd.ps1:2 char:78
+ ... ameter(Mandatory = $false)] [bool] $EnableVMSwitchExtension = $false,
+                                                                          ~
Missing expression after ','.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MissingExpressionAfterToken
 
, message: "At C:\\C\\antrea-agent\\var\\lib\\antrea-windows\\VMSwitchExtension-Antre\r\naAgent-Containerd.ps1:2 char:78\r\n+ ... ameter(Mandatory = $false)] [bool] $EnableVMSwitchExtension = $false,\r\n+                                                                          ~\r\nMissing expression after ','.\r\n    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException\r\n    + FullyQualifiedErrorId : MissingExpressionAfterToken\r\n \r\n"
```

Reply:
Still found FailedPostStartHook error:
```
Warning  FailedPostStartHook  6s    kubelet            Exec lifecycle hook ([powershell -file $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent-Containerd.ps1 -EnableVMSwitchExtension $true]) for Container "antrea-agent" in Pod "antrea-agent-windows-qkvz5_kube-system(34b4dc91-40bd-43fe-98dc-cc6012ed397f)" failed - error: command 'powershell -file $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent-Containerd.ps1 -EnableVMSwitchExtension $true' exited with 1: C:\C\antrea-agent\var\lib\antrea-windows\VMSwitchExtension-AntreaAg
ent-Containerd.ps1 : Cannot process argument transformation on parameter 'EnableVMSwitchExtension'. Cannot convert 
value "System.String" to type "System.Boolean". Boolean parameters accept only Boolean values and numbers, such as 
$True, $False, 1 or 0.
    + CategoryInfo          : InvalidData: (:) [VMSwitchExtensi...-Containerd.ps1], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,VMSwitchExtension-AntreaAgent-Containerd.ps1
 
, message: "C:\\C\\antrea-agent\\var\\lib\\antrea-windows\\VMSwitchExtension-AntreaAg\r\nent-Containerd.ps1 : Cannot process argument transformation on parameter 'EnableVMSwitchExtension'. Cannot convert \r\nvalue \"System.String\" to type \"System.Boolean\". Boolean parameters accept only Boolean values and numbers, such as \r\n$True, $False, 1 or 0.\r\n    + CategoryInfo          : InvalidData: (:) [VMSwitchExtensi...-Containerd.ps1], ParentContainsErrorRecordException\r\n    + FullyQualifiedErrorId : ParameterArgumentTransformationError,VMSwitchExtension-AntreaAgent-Containerd.ps1\r\n \r\n"
  Normal  Pulled  1s  kubelet  Container image "antrea/windows-ovs:1809-3.0.5" already present on machine
```

Reply:
it was because passed boolean value was treated as string. Now fixed and tests passed.